test-site/webpack.config.js

```diff
@@ -20,6 +21,11 @@ module.exports = [{
     path: path.resolve(__dirname, 'dist'),
     libraryTarget: 'window'
   },
+  plugins: [
+    new webpack.DefinePlugin({
```

Comment:
nit: can this be assigned to a variable and then used here and below so the logic isn't repeated?